services/online_exam_service/.env

```
@@ -0,0 +1,18 @@
+# SERVER CONFIG
+PORT=8083
+
+# DATABASE CONFIG (kết nối đến course_db)
+DB_HOST=localhost
+DB_PORT=5433
+DB_USER=postgres
+DB_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)